build.py

```diff
@@ -120,33 +106,34 @@ def clean():
 # Docs
 # ====
 
-def _sphinx_cmd(packages, cmd):
+def _sphinx_cmd(cmd, extra_pkgs=[]):
     envdir = _env()
-    run('pip', 'install', *packages)
+    _deps()
```

Comment:
I'm reading up on Tox. There may be a better way to use tox to manage sphinx ...

https://testrun.org/tox/latest/config.html
https://testrun.org/tox/latest/example/general.html#integrating-sphinx-documentation-checks